lib/atom-html-preview-view.coffee

```diff
@@ -102,10 +102,7 @@ class AtomHtmlPreviewView extends ScrollView
     </html>
     """
     iframe = document.createElement("iframe")
-    #Fix courtesy of github user kwaak (https://github.com/webBoxio/atom-html-preview/issues/1/#issuecomment-49639162)
-    #This allows use of relative scripts and css files
-    iframe.setAttribute("sandbox", "allow-scripts allow-same-origin")
-    iframe.src = @getPath()
+    iframe.src = "data:text/html;charset=utf-8,#{encodeURI(text)}"
```

Comment:
This is a regression; it removes the fix for #1 and #7 committed in ce3634cbb01efd8bdc4923a33ee3916a8db019fe.